blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/ConfigurationTest.java

```diff
@@ -37,31 +40,38 @@ public void tearDown() throws IOException {
 
     @Test
     public void testConfiguration() {
-        Configuration config = Configuration.getInstance();
-        Map<Object, Object> properties = config.getProperties();
-
-        Assert.assertNotNull(properties);
+        try {
+            Configuration config = Configuration.getInstance();
+            Map<Object, Object> properties = config.getProperties();
 
-        Assert.assertEquals("127.0.0.1:19180", config.getStringProperty(CoreConfig.CASSANDRA_HOSTS));
-        System.setProperty("CASSANDRA_HOSTS", "127.0.0.2");
-        Assert.assertEquals("127.0.0.2", config.getStringProperty(CoreConfig.CASSANDRA_HOSTS));
+            Assert.assertNotNull(properties);
 
-        Assert.assertEquals(60000, config.getIntegerProperty(CoreConfig.SCHEDULE_POLL_PERIOD));
+            Assert.assertEquals("127.0.0.1:19180", config.getStringProperty(CoreConfig.CASSANDRA_HOSTS));
+            System.setProperty("CASSANDRA_HOSTS", "127.0.0.2");
```

Comment:
Not a big deal but I would create a constant for "127.0.0.2" if its being used multiple times.